sass/routes/_shortcuts-modal.scss

```diff
@@ -83,20 +83,24 @@
 }
 
 .shortcut-line {
-    margin: 16px 0;
+    padding: 4px;
 
     span {
         &:first-child {
-            margin-right: 5px;
+            margin-top: 4px;
```

Comment:
remove `margin-top:4px`
replace with `margin-left: 0px`

Reply:
resolved

Reply:
> remove `margin-top:4px`
> replace with `margin-left: 0px`

looks like the above did not get applied?